frontend/src/metabase/admin/tasks/containers/Logs.jsx

```diff
@@ -113,7 +113,9 @@ export default class Logs extends Component {
     const renderedLogs = filteredLogs.map(ev => {
       const timestamp = moment(ev.timestamp).format();
       const uuid = ev.process_uuid || "---";
-      return `[${uuid}] ${timestamp} ${ev.level} ${ev.fqns} ${ev.msg}`;
+      return `[${uuid}] ${timestamp} ${ev.level} ${ev.fqns} ${ev.msg}${
+        ev.exception ? "\n" + ev.exception : ""
```

Comment:
one complaint that *might* come up is that stack traces are typically formatted / indented - we won't be able to (trivially) do that here, but it's still better than what we had before!

Reply:
It turns out the stacktrace came back indented. I tweaked the css so the page displays it.